src/webgpu/util/texture/texture_ok.spec.ts

```diff
@@ -0,0 +1,162 @@
+export const description = 'checkPixels helpers behave as expected against real textures';
+
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { GPUTest } from '../../gpu_test.js';
+
+import { TexelView } from './texel_view.js';
+import { textureContentIsOKByT2B } from './texture_ok.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('float32')
+  .desc(`Basic test that actual/expected must match, for float32.`)
+  .params(u =>
+    u
+      .combineWithParams([
+        { format: 'rgba32float' }, //
+        { format: 'rg32float' },
+      ] as const)
+      .beginSubcases()
+      .combineWithParams([
+        // Expected data is 0.6 in all channels
+        { data: 0.6, opts: { maxFractionalDiff: 0 }, _ok: true },
```

Comment:
Known failing because I removed quantization of the expected float value to the texel format, so they don't match exactly. Unsure whether I should reintroduce the quantization, but I think I shouldn't. maxDiffULPsForFloatFormat: 1 is the reasonable thing to do here, this is just a unittest.

Reply:
sounds ok to me